src/rl/models/value_net.py

```
@@ -0,0 +1,137 @@
+
+    def forward(
+            self,
+            node_embeddings: torch.Tensor,
+            node_mask: torch.Tensor,
+            batch: Optional[torch.Tensor] = None,
+    ) -> torch.Tensor:
+        
+        if batch is not None:
+            pooled = self.pool(node_embeddings, batch) # node_embeddings: (num_nodes, hidden_dim), batch: (num_nodes,)
+        else:
+            batch_size, num_nodes, hidden_dim = node_embeddings.size() # (batch_size, num_nodes, hidden_dim)
```

Ruff (0.14.1)

Unpacked variable `hidden_dim` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)